src/core/quantum_gate.jl

```diff
@@ -1744,7 +2024,7 @@ get_operator(gate::PhaseShift,T::Type{<:Complex}=ComplexF64) = phase_shift(gate.
 
 Base.inv(gate::PhaseShift) = phase_shift(gate.target, -gate.phi)
 
-get_gate_parameters(gate::PhaseShift)=Dict("phi" =>gate.phi)
+get_gate_parameters(gate::PhaseShift)=Dict("lambda" =>gate.phi)
```

Comment:
inconsistent key name and internal field, why did we change it?

Reply:
@plerouxanyon we should rename the internal `gate.phi `to `gate.lambda`

Reply:
We changed it in https://github.com/anyonlabs/Snowflake.jl/pull/261.  Our quantum computer expects an "rz" symbol with a "lambda" parameter. The problem is that we are coupling the gate definition with the textual symbol that we send to Anyon's quantum computer. We should be able to use `PhaseShift` with a `phi` parameter and convert that to `rz` with a `lambda` parameter only when we use `AnyonQPU`. I will make a follow-up issue for this.

Reply:
I've created #282 to address the issue.